packages/unless/src/index.ts

```diff
@@ -0,0 +1,89 @@
+import type { Request, Response, NextFunction } from '@tinyhttp/app'
+
+//Options for the middleware
+export interface UnlessMiddlewareOptions {
```

Comment:
Not sure why it is an interface. It only contains properties, doesn't extend anything and nothing implements it. I'd use `type` instead, wrapped in the `Partial<T>`